### PR TITLE
Updated authentication/authorization logs in syslog format

### PR DESCRIPTION
**Pull request by amar-shalgar**
Updated the authentication and authorization logs in syslog format
Ran go tools

### CODE REVIEW

File: lib-utilities/services/auth.go [hunk 1]
```diff
@@ -59,6 +58,9 @@ func IsAuthorized(sessionToken string, privileges, oemPrivileges []string) errRe
 	if response.StatusCode == http.StatusServiceUnavailable {
 		msgArgs = append(msgArgs, fmt.Sprintf("%v:%v", config.Data.DBConf.InMemoryHost, config.Data.DBConf.InMemoryPort))
 	}
+	// getting user details
```
Comment (jeevan-kamkar) [inline lib-utilities/services/auth.go]: remove the commented line

File: svc-api/router/router.go [hunk 2]
```diff
@@ -19,13 +19,15 @@ package router
 import (
 	"bytes"
 	"encoding/json"
-	"fmt"
+	//"fmt"
```
Comment (jeevan-kamkar) [inline svc-api/router/router.go]: remove the commented code

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the minor commnets

File: lib-utilities/services/auth.go (hunk 1 above)
Comment (amar-shalgar) [reply lib-utilities/services/auth.go]: Addressed the comments

File: svc-api/router/router.go (hunk 2 above)
Comment (amar-shalgar) [reply svc-api/router/router.go]: Addressed the comments

**Review by jeevan-kamkar — APPROVED** [review]

**Review by irappa-pattar — APPROVED** [review]

**Review by hemkapoor — APPROVED** [review]

**Pull request merged**